doc/classes/VisualShaderNodeCustom.xml

```diff
@@ -0,0 +1,138 @@
+			<argument index="0" name="port" type="int">
+			</argument>
+			<description>
+				By overriding this function, the returned string will be used as a name for associated input [code]port[/code].
+				Does not required to be overrided (but usually desired). If its not overrided, all input ports will get names: [code]in + portIndex[/code].
+			</description>
+		</method>
+		<method name="_get_input_port_type" qualifiers="virtual">
+			<return type="int" enum="VisualShaderNode.PortType">
+			</return>
+			<argument index="0" name="port" type="int">
+			</argument>
+			<description>
+				By overriding this function, the returned type (see [constant VisualShaderNode.PortType]) will be used for associated input [code]port[/code].
+				Does not required to be overrided (but usually desired). If its not overrided, all input ports will return [constant VisualShaderNode.PORT_TYPE_SCALAR] type.
+			</description>
+		</method>
+		<method name="_get_name" qualifiers="virtual">
+			<return type="String">
+			</return>
+			<description>
+				By overriding this function, name of the associated custom node in the members dialog (and graph) will be changed to the returned value.
+				Does not required to be overrided (but usually desired). If its not overrided, the custom node will be named as "Unnamed".
+			</description>
+		</method>
+		<method name="_get_output_port_count" qualifiers="virtual">
+			<return type="int">
+			</return>
+			<description>
+				By overriding this function, the output port amount is changed to the returned value.
+				Required to be overrided in most cases (by default, no output ports are exist).
+			</description>
+		</method>
+		<method name="_get_output_port_name" qualifiers="virtual">
+			<return type="String">
+			</return>
+			<argument index="0" name="port" type="int">
+			</argument>
+			<description>
+				By overriding this function, the returned string will be used as a name for associated output [code]port[/code].
+				Does not required to be overrided (but usually desired). If its not overrided, all output ports will get names: [code]out + portIndex[/code].
+			</description>
+		</method>
+		<method name="_get_output_port_type" qualifiers="virtual">
+			<return type="int" enum="VisualShaderNode.PortType">
+			</return>
+			<argument index="0" name="port" type="int">
+			</argument>
+			<description>
+				By overriding this function, the returned type (see [constant VisualShaderNode.PortType]) will be used for associated output [code]port[/code].
+				Does not required to be overrided (but usually desired). If its not overrided, all output ports will return [constant VisualShaderNode.PORT_TYPE_SCALAR] type.
+			</description>
+		</method>
+		<method name="_get_return_icon_type" qualifiers="virtual">
+			<return type="int" enum="VisualShaderNode.PortType">
+			</return>
+			<description>
+				By overriding this function, return icon of the associated custom node in the members dialog will be changed to the returned type.
```

Comment:
Proofreading:
```
Override this method to define the return icon of the associated custom node in the Visual Shader Editor's members dialog.
Defining this method is [b]optional[/b]. If not overridden, the return icon will be of type [constant VisualShaderNode.PORT_TYPE_SCALAR].
```

Reply:
Nope, it will be empty

Reply:
"If not overridden, the return icon will not be placed" - I guess this ok ?

Reply:
Maybe "If not overridden, no return icon is shown."

Reply:
Ok, fixed